Formula/ladspa-sdk.rb

```diff
@@ -0,0 +1,46 @@
+class LadspaSdk < Formula
+  desc "Linux Audio Developer's Simple Plugin"
+  homepage "https://ladspa.org"
+  url "https://www.ladspa.org/download/ladspa_sdk_1.15.tgz"
+  sha256 "4229959b09d20c88c8c86f4aa76427843011705df22d9c28b38359fd1829fded"
+  license "LGPL-2.1-or-later"
```

Comment:
Looking at upstream `doc/license.html`, shouldn’t that be `LGPL-2.1-only`?

Reply:
Hmm `src/ladspa.h` says `LGPL-2.1-or-later` and there's also this in `doc/license.html`:

> Please note that this is not intended to be the final license for LADSPA. In the long term it is hoped that LADSPA will have a public license that is even less restrictive, so that commercial applications can use it without having to use a derived LGPL library (in a way that still protects the open-source community). It may be that LGPL is already free enough for this, but we aren't sure.

I'm guessing `LGPL-2.1-or-later` instead of `LGPL-2.1-only` would fit with their goal to be less restrictive

Reply:
Even though I agree with you in spirit, we should still assume that we’re bound by `doc/license.html` just the same. That means we’re bound by both `LGPL-2.1-or-later` and `LGPL-2.1-only` at the same time, which can be simplified to `LGPL-2.1-only`.